nixos/modules/services/printing/cupsd.nix

```diff
@@ -185,6 +190,17 @@ in
         '';
       };
 
+      printersConf = mkOption {
+        type = types.nullOr types.lines;
+        default = null;
+        description = ''
+          Contents of the printers configuration file of the CUPS daemon
+          (<filename>printers.conf</filename>). When left null, cups will use
+          the (<filename>/etc/cupsd/printers.conf</filename>) as the configuration
```

Comment:
@globin, how do you feel about porting this to 17.03? This is a feature but seems a very small one.